lib/internal/Magento/Framework/Mview/View/Subscription.php

```diff
@@ -193,11 +191,23 @@ protected function getLinkedViews()
      * Build trigger statement for INSERT, UPDATE, DELETE events
      *
      * @param string $event
-     * @param \Magento\Framework\Mview\View\ChangelogInterface $changelog
+     * @param \Magento\Framework\Mview\ViewInterface $view
      * @return string
      */
-    protected function buildStatement($event, $changelog)
+    protected function buildStatement($event, $view)
     {
+        // Get the subscription for the specific view and specific table.
+        // We will use column name from it.
+        $subscriptions = $view->getSubscriptions() ?? [];
```

Comment:
I guess since the goal here is to get the `$subscription['column']` I would encapsulate this logic in a private function, at least it keeps the buildStatement a little simpler.